nrf5/hal/hal_temp.c

```diff
@@ -25,18 +25,36 @@
  */
 
 #include <stdio.h>
+#include <stdint.h>
 #include "mphalport.h"
 #include "hal_temp.h"
 
+#if BLUETOOTH_SD
```

Comment:
it has to.. as it must use the softdevice version of temp access. It cannot go directly on the peripheral. The other solution is to make a HAL-hal.